fixes #1142 - switch import directory mode to honor neo4j import dire… (#61)
larusba/neo4j-apoc-procedures: docs/asciidoc/config.adoc

Set these config options in `$NEO4J_HOME/neo4j.conf`

> All these follow boolean options default to **false**, i.e. they are disabled, unless mentioned otherwise.

Reviewer: ```suggestion
All boolean options have **false** as default value, i.e. they are disabled, unless mentioned otherwise.
```
